projects/swimlane/ngx-datatable/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@swimlane/ngx-datatable",
+  "version": "15.0.2",
+  "description": "ngx-datatable is an Angular table grid component for presenting large and complex data.",
+  "peerDependencies": {
```

Comment:
I believe you have to add the dependencies as well to the library's package.json, right?

Reply:
Not unless the library has any dependencies, afaik.